langserver/server.go

```diff
@@ -164,6 +169,30 @@ func (s *server) connectPrometheus(url string) error {
 			return err
 		}
 
```

Comment:
Let's assume version 2.13 in case we get a 404 error. (And add a comment why this hack is in place).

Reply:
https://github.com/prometheus-community/promql-langserver/pull/122#issuecomment-598702078